omod/src/main/webapp/resources/scripts/visit/visit.js

```diff
@@ -817,8 +817,8 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                         $scope.encounterDateFormat = sameDate($scope.visit.startDatetime, $scope.visit.stopDatetime) ? "hh:mm a" : "hh:mm a (d-MMM)";
 
                         if ($scope.suppressActions !== true) {
-                          AppFrameworkService.getUserExtensionsFor("patientDashboard.visitActions").then(function (ext) {
-                            $scope.visitActions = ext;
+                          CoreappsService.getUserExtensionsFor("patientDashboard.visitActions", $scope.patientUuid).then(function (ext) {
+                              $scope.visitActions = ext;
```

Comment:
One thing here - this only takes in the patient, but shouldn't this also take in the visit optionally?  Without the visit, this will always return actions based on the current active visit.  But if we view a "past visit" within the visit note, shouldn't the actions that are available be based on that past visit and not on the currently active visit?
@cioan / @mogoodrich ?

Reply:
@mseaton , then we would need to modify the REST controller to accept the visit parameter?
https://github.com/openmrs/openmrs-module-coreapps/blob/master/omod/src/main/java/org/openmrs/module/coreapps/web/controller/CoreappsRestController.java#L92

Reply:
Presumably, yes, as well as the appcontextmodelgenerator.  Maybe this is a separate ticket, we should discuss.

Reply:
@mseaton , I have added the visit parameter to the REST request